drivers/at_cmd/Kconfig

```diff
@@ -12,6 +12,10 @@ config AT_CMD
 
 if AT_CMD
 
+config AT_CMD_SYS_INIT
+	bool "Initialize the AT command driver during system init"
+	default y if BSD_LIBRARY_SYS_INIT
+
 config AT_CMD_INIT_PRIORITY
```

Comment:
This config should not be available in KConfig when `AT_CMD_SYS_INIT=n`
Could you put it in a guard or make it depend on `AT_CMD_SYS_INIT`?

Reply:
Good point, fixed.